### ISSUE TITLE
The range of importance scores is not between -1 and 1 in skrebate 0.3-0.4

### ISSUE THREAD

**Issue by weixuanfu** [EpistasisLab/scikit-rebate#33]
I tested the codes below with skrebate 0.4
```
import pandas as pd
import numpy as np
from sklearn.pipeline import make_pipeline
from skrebate import ReliefF

genetic_data = pd.read_csv('https://github.com/EpistasisLab/scikit-rebate/raw/master/data/'
                           'GAMETES_Epistasis_2-Way_20atts_0.4H_EDM-1_1.tsv.gz',
                           sep='\t', compression='gzip')

features, labels = genetic_data.drop('class', axis=1).values, genetic_data['class'].values


fs = ReliefF()
fs.fit(features, labels )

for feature_name, feature_score in zip(genetic_data.drop('class', axis=1).columns,
                                       fs.feature_importances_):
    print(feature_name, '\t', feature_score)
```

And stdout is showed below, which is not similar with the [example](https://epistasislab.github.io/scikit-rebate/using/#acquiring-feature-importance-scores)

```
N0       -24.5
N1       -396.5
N2       -753.0
N3       -640.0
N4       -614.5
N5       -668.5
N6       -810.5
N7       -767.5
N8       -486.0
N9       -637.5
N10      -47.5
N11      -666.0
N12      -553.5
N13      -253.0
N14      -254.0
N15      -816.5
N16      -564.0
N17      -632.0
P1       9543.0
P2       9782.5
```

Then I downgraded to skrebate 0.2 and tested with the same scripts above. The stdout was changed to:
```
N0       -0.00025
N1       -0.00521875
N2       -0.0086125
N3       -0.00770625
N4       -0.00755
N5       -0.00968125
N6       -0.01015625
N7       -0.00959375
N8       -0.00599375
N9       -0.0082625
N10      -0.001125
N11      -0.00750625
N12      -0.00778125
N13      -0.00265625
N14      -0.00309375
N15      -0.0104125
N16      -0.007375
N17      -0.00760625
P1       0.1201375
P2       0.1222
```

@ryanurbs  and I think it maybe a normalization issue.

**Comment by weixuanfu** [comment]
[Codes for averaging the scores in 0.2](https://github.com/EpistasisLab/scikit-rebate/blob/8567aca24d0eb84aacf6563ac748c73e6bb04742/skrebate/relieff.py#L433-L438)

I think we need restore these codes in latest version of skrebate, @rhiever @ryanurbs ?

**Comment by weixuanfu** [comment]
I add a commit for fixing this issue in [one of my branches](https://github.com/weixuanfu/scikit-rebate/tree/ave_score). And then the output is:

```
N0       -0.000153125
N1       -0.002478125
N2       -0.00470625
N3       -0.004
N4       -0.003840625
N5       -0.004178125
N6       -0.005065625
N7       -0.004796875
N8       -0.0030375
N9       -0.003984375
N10      -0.000296875
N11      -0.0041625
N12      -0.003459375
N13      -0.00158125
N14      -0.0015875
N15      -0.005103125
N16      -0.003525
N17      -0.00395
P1       0.05964375
P2       0.061140625
```

Is this a correct implementation?

**Comment by rhiever** [comment]
We should double-check with @ryanurbs, but I don't think it's vital for the scores to be scaled between [-1, 1] because they're all relative scores anyway.

**Comment by weixuanfu** [comment]
I understand that, maybe we need update examples in docs.

**Comment by rhiever** [comment]
@ryanurbs clarified privately to me that the common expectation for ReliefF is to have the scores in the range of [-1, 1]. He said he will work with you to properly (re-)implement the normalization procedure.

**Comment by weixuanfu** [comment]
I closed this issue since the problem is solved in dev branch
